go.sum

```diff
@@ -205,6 +205,8 @@ github.com/spiffe/go-spiffe v0.0.0-20170907221946-2bb3101d62b4/go.mod h1:q0X9/v7
 github.com/spiffe/go-spiffe v0.0.0-20190717182101-d8657cb50cae/go.mod h1:HyNeJnVYkDyQgB2qcSPxVYkAA2F3lQu51bDxNpFcKxY=
 github.com/spiffe/go-spiffe v0.0.0-20190922191205-018e7197ed1c h1:wpwh25WjvKF8/+N+wMy1u9nMiOXfw5sqpmL5ZSAFIWU=
 github.com/spiffe/go-spiffe v0.0.0-20190922191205-018e7197ed1c/go.mod h1:HyNeJnVYkDyQgB2qcSPxVYkAA2F3lQu51bDxNpFcKxY=
+github.com/spiffe/spiffe-helper v0.0.0-20191008190511-e11122b96bb9 h1:4Ze5T6YLet2KlPoGXCS0yshA/zv/EkJmirbKUL1PQ08=
```

Comment:
This change should be reverted once you rename the module back to github.com/spiffe/spiffe-helper

Reply:
done